src/Infrastructure/Services/UserService.cs

```diff
@@ -421,6 +422,14 @@ public async Task<Response<string>> RespondToMfaChallengeAsync(string mfaCode, s
 
         private string CleanPhoneNumber(string phoneNumber)
         {
+            // ensure its a valid uk mobile using regex
+            var isValid = Regex.IsMatch(phoneNumber, @"^((\+44\s?|0)7([45789]\d{2}|624)\s?\d{3}\s?\d{3})$",
+                RegexOptions.None, TimeSpan.FromMilliseconds(100));
+            if (!isValid)
+            {
+                throw new ValidationException();
```

Comment:
A message in the exception constructor would be useful here, e.g. "'xyz' is not a valid mobile phone number."